cinemy/cinemy/urls.py

```diff
@@ -24,5 +24,5 @@
     path("", front_page, name="front_page"),
     path("contact/", include("contact.urls")),
     path("user/", include("login.urls")),
-    path("book_ticket/", include("cinema.urls")),
+    path("", include("cinema.urls")),
```

Comment:
and here I advise you again to use signifiant prefixes to the app's urls. Usually, if a prefix is not a good fit for the all the urls included, then you can split the app into more apps for modularity. Now you can leave it as it is, but consider this in the future

Reply:
Will rework this in the future